src/p4est_to_p8est.h

```diff
@@ -131,6 +142,10 @@
 #define p4est_wrap_leaf_t               p8est_wrap_leaf_t
 #define p4est_wrap_flags_t              p8est_wrap_flags_t
 #define p4est_vtk_context_t             p8est_vtk_context_t
+#define p4est_file_context              p8est_file_context
```

Comment:
We usually get away without redefining the names of struct (see rest of p4est).

Reply:
If I remove the redefinition I get a compiler error:

```
../p4est/src/p4est_to_p8est.h:145:41: error: invalid application of ‘sizeof’ to incomplete type ‘p8est_file_context_t {aka struct p8est_file_context}’
 #define p4est_file_context_t            p8est_file_context_t
                                         ^
../p4est/src/p4est_base.h:200:65: note: in definition of macro ‘P4EST_ALLOC’
                                                    (n) * sizeof(t))
                                                                 ^
../p4est/src/p4est_io.c:568:53: note: in expansion of macro ‘p4est_file_context_t’
   p4est_file_context_t *file_context = P4EST_ALLOC (p4est_file_context_t, 1);
```

Reply:
I see. Please solve the problem in the same way as with the VTK context: use an `#ifdef` inside the `.c` file for the redefinition.

Reply:
Ah, I see. The redefinition in the `.c` is now implemented.